### PR TITLE
Minor changes has been done to pipelines_intro_kfp

### PR DESCRIPTION
**Pull request by bhupana**
Minor changes has been done to pipelines_intro_kfp

If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/CODEOWNERS) file under the `# Community Notebooks` section, pointing to the author or the author's team.
- [x] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/docs/contributing.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/pipelines/pipelines_intro_kfp.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (andrewferlitsch) [inline notebooks/official/pipelines/pipelines_intro_kfp.ipynb:8]: <p>BUCKET_NAME = "[your-bucket-name]"&nbsp;# @param {type:"string"}</p><p>BUCKET_URI = f"gs://{BUCKET_NAME}"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/pipelines_intro_kfp.ipynb:8]: <p>delete this cell. Doesn't seem to make sense or belong here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (andrewferlitsch) [inline notebooks/official/pipelines/pipelines_intro_kfp.ipynb:8]: <p>please use this form for the service account:</p><p> </p><p>if (</p><p>&nbsp;&nbsp;SERVICE_ACCOUNT == ""</p><p>&nbsp;&nbsp;or SERVICE_ACCOUNT is None</p><p>&nbsp;&nbsp;or SERVICE_ACCOUNT == "[your-service-account]"</p><p>):</p><p>&nbsp;&nbsp;# Get your GCP project id from gcloud</p><p>&nbsp;&nbsp;shell_output = !gcloud auth list 2&gt;/dev/null</p><p>&nbsp;&nbsp;SERVICE_ACCOUNT = shell_output[2].replace("*", "").strip()</p><p>&nbsp;&nbsp;print("Service Account:", SERVICE_ACCOUNT)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

**Comment by bhupana** [comment]
worked on revied comments please check

**Review by andrewferlitsch — APPROVED** [review]

**Pull request merged**